views/accessibility-statement.html

```diff
@@ -18,120 +18,87 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
-        <h1 class="govuk-heading-l">
-            Accessibility statement for the Remove a company director service
+        <h1 class="govuk-heading-xl">
+            {{ i18n.accessibilityStatementHeading }}
         </h1>
 
-        <ul class="govuk-list govuk-list--bullet">
-            <li><a class="govuk-link" href="#about-service">About this service</a></li>
-            <li><a class="govuk-link" href="#how-accessible">How accessible this service is</a></li>
-            <li><a class="govuk-link" href="#reporting">Reporting accessibility problems with this service</a></li>
-            <li><a class="govuk-link" href="#enforcement">Enforcement procedure</a></li>
-            <li><a class="govuk-link" href="#technical-information">Technical information about this service’s accessibility</a></li>
-            <li><a class="govuk-link" href="#non-accessible-content">Non-accessible content</a></li>
-            <li><a class="govuk-link" href="#were-doing">What we’re doing to improve accessibility</a></li>
-            <li><a class="govuk-link" href="#preparation-of-statement">Preparation of this accessibility statement</a></li>
-        </ul>
-
-        <h2 class="govuk-heading-m" id="about-service">About this service</h2>
-        <p class="govuk-body">This accessibility statement applies to the Remove a company director service.</p>
-
-        <p class="govuk-body">This service is run by <a class="govuk-link" href="https://www.gov.uk/government/organisations/companies-house">Companies House</a>. We want as many people as possible to be able to use this service. For example, that means you should be able to:</p>
+        <p class="govuk-body">{{ i18n.accessibilityStatementAboutPreLink }}<a class="govuk-link" href="https://www.gov.uk/government/organisations/companies-house">{{ i18n.accessibilityStatementAboutLinkText }}</a>{{ i18n.accessibilityStatementAboutPostLink }}</p>
```

Comment:
external href value language parameter? Same for other external links if required

Reply:
I can add a language param like we have elsewhere: `https://www.gov.uk/government/organisations/companies-house?lang={{lang|default('en')}}`
I don't believe this will work for this or any of the other external links on this page however. The welsh version of this specific link would be: `https://www.gov.uk/government/organisations/companies-house.cy`
Are we handling these kind of language specific links elsewhere?

Reply:
ch related links example for .cy is on startpage

Reply:
Ah, I actually tried this approach but having .en on the end of the url seems to give a page not found error. This seems to also be an issue on the start page link. 
<img width="1108" alt="Screenshot 2024-03-20 at 17 15 37" src="https://github.com/companieshouse/officer-filing-web/assets/117649383/c528f006-f074-40da-9fa6-f2bb797675b9">

Reply:
Yes the page doesn't expect parameter for english (similar to start page)

Reply:
Resolved in line with the fixed start page code